### DISCUSSION TITLE
Can i use local embedding model? How?

### DISCUSSION THREAD

**Discussion by HuaC-Z** [langgenius/dify#4952] [General]
After the project is started and the embedding model is not configured, a message will appear: Default model not found for ModelType.TEXT_EMBEDDING

**Comment by patryk20120** [comment]
You can use only predefined providers and LLMs. Go to Settings -> Model Providers to see supported providers.

**Comment by santime27** [comment]
yes, actually I´m running Ollama and there I´m running "mxbai-embed-large" model to do embeddings. You can add this functionally in "Settings - Model Provider" and add Ollama, in these config window, add the model name, and ollama api access
